test/unit/test_models.py

```diff
@@ -42,6 +42,20 @@ def test_missing_inclusion_proof(self):
                 inclusion_promise=None,
             )
 
+    def test_missing_inclusion_promise_round_trip(self, signing_bundle):
+        """
```

Comment:
Shouldn't this use a bundle without an inclusion promise and test with that? I'm guessing deserializing that bundle will still fail, based on looking at `Bundle._verify()`.  Maybe you are planning to fix that in a future PR?  It would make sense to test a complete bundle round trip here but let me know what you think...

I think signing with current staging and just manually removing the inclusion promise (and maybe also integration time) from the bundle file should work?

Reply:
scratch that, I didn't read that fully